### DISCUSSION TITLE
Way to stop space insertion after autocomplete

### DISCUSSION THREAD

**Discussion by PeterCLarsen** [Benature/obsidian-text-format#87] [Q&A]
I really enjoy using text-format, but the one annoying feature is the automatic insertion of a space after you hit enter to autocomplete the word. If the word is at the end of a sentence or you want to insert a punctuation mark, you must backspace. Using backspace, instead of the space bar, for me at least, slows down my workflow. Mostly, because I didn't realise the space was inserted and had to go back later to fix it later.

Is there a setting I can adjust to change this behaviour?

**Comment by Benature** [comment]
Hi, the auto complete behavior belongs to an obsidian plugin or the system keyboard (win? Mac? iOS? Android?)? For me using iOS, it indeed add a space every time auto complete. But if I add a punctuation, it will auto remove the blank space.
